client/layout/transient-notices/snackbar/index.js

```diff
@@ -73,11 +108,11 @@ function Snackbar(
 		<div
 			ref={ ref }
 			className={ classes }
-			onClick={ onRemove }
+			onClick={ ! explicitDismiss ? dismissMe : noop }
 			tabIndex="0"
-			role="button"
-			onKeyPress={ onRemove }
-			aria-label={ __( 'Dismiss this notice' ) }
+			role={ ! explicitDismiss ? 'button' : '' }
+			onKeyPress={ ! explicitDismiss ? dismissMe : noop }
+			aria-label={ ! explicitDismiss ? __( 'Dismiss this notice' ) : '' }
```

Comment:
Missing text domain here.

Reply:
Interesting, good catch. This is forked from the component in Gutenberg and has been [pushed upstream](https://github.com/WordPress/gutenberg/pull/26952) and the fork will be [removed](https://woostartp2.wordpress.com/2020/11/10/forking-the-snackbar-component-from-gutenberg/) once the changes become available to us via Gutenberg. So, I don't _think_ there is much point in changing this - I imagine that this is correctly translated in Gutenberg.